source/components/ui/msm-modal/msm-modal.scss

```diff
@@ -0,0 +1,236 @@
+@media (min-width: 768px) {
+  .modal-mobile {
+    display: none;
+  }
+
+  .modal-backdrop {
```

Comment:
Maybe we should introduce some media mixins. Like those: https://gist.github.com/imjared/9b59c2db295acaefaac5. What do you think? Would be easier to compare the differences between the break points.
```scss
.modal-backdrop {
  &.in {
    @include breakpoint-max(767px) { // isn't this unnecessary?
      opacity: 1;
    }
    @include breakpoint-min(768px) {
      opacity: 0.65;
    }
  }
}
```